logux-error/index.test.js

```diff
@@ -10,6 +10,13 @@ function catchError (desc, type, received) {
   return error
 }
 
+it('do not crash if captureStackTrace does not exist', () => {
```

Comment:
can you change `do` to `does` to have `it does not…` sentence?

Reply:
Yes, sure. Good point. I've updated the commit and push with --force.